src/theme.c

```diff
@@ -853,6 +854,9 @@ entry(struct theme *theme, const char *key, const char *value)
 	if (match_glob(key, "magnifier.filter")) {
 		theme->mag_filter = atoi(value);
 	}
+	if (match_glob(key, "magnifier.increment")) {
+		theme->mag_increment = atof(value);
```

Comment:
Please use `set_float(value, theme->mag_increment)` here, that one is locale aware.

Reply:
Thanks - didn't know about that function!